### PR TITLE
include config dir clearance in clean-app script

### PR DESCRIPTION
**Pull request by mallexxx**
Task/Issue URL: https://app.asana.com/0/1201037661562251/1208943073994046/f

**Description**:
- Include `~/Library/Group\ Containers/*.com.duckduckgo.macos.browser.app-configuration.debug/*` path to clear by the `./clean-app.sh` script

**Optional E2E tests**:
- [ ] Run PIR E2E tests
	Check this to run the Personal Information Removal end to end tests. If updating CCF, or any PIR related code, tick this.

**Steps to test this PR**:
1. Validate downloaded configs directory is cleared by the script for both DMG and AppStore builds

**Definition of Done**:

* [ ] Does this PR satisfy our [Definition of Done](https://app.asana.com/0/1202500774821704/1207634633537039/f)?

---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)

### CODE REVIEW

File: clean-app.sh [hunk 1]
```diff
@@ -8,27 +8,32 @@ delete_data() {
 	if defaults read "${bundle_id}" &>/dev/null; then
 		defaults delete "${bundle_id}"
 	fi
-	rm -rf "${HOME}/Library/Containers/${bundle_id}/Data"
+	rm -r "${HOME}/Library/Containers/${bundle_id}/Data"
```
Comment (ayoy) [inline clean-app.sh:11]: Why are we removing `-f`? This will break the script when the directory doesn't exist.

File: clean-app.sh [hunk 2]
```diff
@@ -38,6 +43,21 @@ esac
...
+    find "${HOME}/Library/Group Containers/" \
+        -type d \
+        -maxdepth 1 \
+        -name "${config_ids}" \
+        -exec basename {} \;
+)
+for config_id in "${config_bundle_ids[@]}"; do
+    path="${HOME}/Library/Group Containers/${config_id}"
+    printf '%s' "Deleting data at ${path}... "
+    rm -r "${path}"
```
Comment (ayoy) [inline clean-app.sh:57]: And same here. A general rule for scripts is that they should do things forcibly unless they also provide error handling.

File: clean-app.sh (hunk 1 above)
Comment (mallexxx) [reply clean-app.sh:11]: well, it does for me; 
but it does report a missing dir status instead if you run it twice

**Review by ayoy — APPROVED** [review]
Looks and works great, thanks @mallexxx!

File: clean-app.sh (hunk 1 above)
Comment (ayoy) [reply clean-app.sh]: Sorry about that, the script actually goes on (we’re not using `set -e` here, but displays an error). Would be great to have it handled gracefully if we’re skipping `-f`. Thanks for updating it.

**Pull request merged**